charts/newrelic-infrastructure/Chart.yaml

```diff
@@ -8,7 +8,7 @@ sources:
   - https://github.com/newrelic/nri-kubernetes/tree/main/charts/newrelic-infrastructure
   - https://github.com/newrelic/infrastructure-agent/
 
-version: 3.12.0
+version: 3.12.1
 appVersion: 3.6.0
```

Comment:
If we make changes to the agent, I believe we want to bump both the `appVersion` and the chart `version`. 

That said, I am new to helm charts and the `nri-kubernetes` release process, so I'm less clear on how this works in practice. 😅  Do we want to make these chart changes in this PR or wait until we do a release next Wednesday?

Reply:
@nserrino I made several updates to address the lints, including the version change, after you approval. :) So pls review the PR again. I will revert the version change if it is not proper.

Reply:
I reverted the chart change before the PR merge